go.mod

```diff
@@ -180,16 +180,16 @@ replace (
 	github.com/prometheus/client_golang => github.com/prometheus/client_golang v1.12.1 // keep this value in sync with sigs.k8s.io/controller-runtime
 	golang.org/x/crypto => golang.org/x/crypto v0.0.0-20220516162934-403b01795ae8
 	google.golang.org/grpc => google.golang.org/grpc v1.40.0 // keep this value in sync with k8s.io/apiserver
```

Comment:
This need to be in sync with https://github.com/kubernetes/apiserver/blob/bdbf5341a8de58461fe16315330e7b67fbac520d/go.mod#L38